### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by LappleApple**

### CODE REVIEW

**Comment by LappleApple** [comment]
For this section: "A goroutine per deployment instance will:" -- are those steps in order, or are they just bullet points?

Also: "Read application secrets from secret/<marathon-appID>" -- broken link? broken code formatting?

**Comment by szuecs** [comment]
> For this section: "A goroutine per deployment instance will:" -- are those steps in order, or are they 
> just bullet points?

These steps are ordered.

> Also: "Read application secrets from secret/" -- broken link? broken code formatting?

It is not broken. 
"secret/" is a path to a specific configured bucket for shared secrets in vault.
For example "cubbyhole/" is another path to another bucket.

File: README.md [hunk 1]
```diff
@@ -42,94 +46,75 @@ godep go install -tags zalando -ldflags "-X main.Buildstamp=`date -u '+%Y-%m-%d_
...
+
+Howler users will vary in their backend-related needs. One approach is to mix different backends; another is to implement a greater number of backends. 
 
-Different users of Howler might have different needs in terms of backends. This could mean using a mix of different backends or implementing more. 
-To allow composability, we choose a "compilation over configuration" approach: to configure your set of backends you have to define a backend configuration similar to [this one](backendconfig/zalando.go) and select it at compile time. 
-All you have to do is the following: 
-- write a [backend config](backendconfig/zalando.go) with the appropiate [build tag](https://golang.org/pkg/go/build/)
-- use build tags to select your configuration: ```godep go install -tags YOUR_TAG github.com/zalando-techmonkeys/howler/...```
+To allow composability, one can choose a "compilation over configuration" approach:
```
Comment (Raffo) [inline README.md]: "one can choose" sounds a little bit wrong as we already choose this approach and there's no way for the user to skip that if they want to configure a different mix of backends. I would stick with the previous "we chose a compilation over configuration approach"

File: README.md [hunk 2]
```diff
@@ -42,94 +46,75 @@ godep go install -tags zalando -ldflags "-X main.Buildstamp=`date -u '+%Y-%m-%d_
...
-the most feature complete F5 RESTful API, and
-[chimp](https://github.com/zalando-techmonkeys/chimp), a PAAS
-style deployment tool:
+Have a look at the [dummy backend](backend/dummy.go) for an example.
+
+####Load Balancing
+[F5](https://f5.com/) produces hardware load balancers like [LTM Big-IP](https://f5.com/products/modules/local-traffic-manager) and [GTM](https://f5.com/products/modules/global-traffic-manager), a smart DNS server.
+
+This diagram shows how to combine LTM Big-IP and GTM DNS server integration with [baboon-proxy](https://github.com/zalando-techmonkeys/baboon-proxy) (currently the most feature-complete F5 RESTful API available) and
+[chimp](https://github.com/zalando-techmonkeys/chimp), a PAAS-style deployment tool:
```
Comment (Raffo) [inline README.md]: I would remove the line about chimp, not sure how it would be useful in this section.

File: README.md [hunk 3]
```diff
@@ -42,94 +46,75 @@ godep go install -tags zalando -ldflags "-X main.Buildstamp=`date -u '+%Y-%m-%d_
...
-secrets.
+This diagram illustrates the steps of secret distribution, and the roles of Howler and other components:
+![Secret distribution integration](https://raw.githubusercontent.com/zalando-techmonkeys/howler/master/docs/secrets-distribution-vault.png)
 
-Howler can help you to provide a new deployed instance with it's
-secrets maintained by [vault](https://github.com/hashicorp/vault).
+#####Requirements
+To use Vault with Howler, you need:
+- a running and unsealed vault
+- "secret" and "cubbyhole" Vault backends
```
Comment (Raffo) [inline README.md]: @szuecs this should come by default, right, is there a way to remove them? If not, it's not a requirement.

**Comment by Raffo** [comment]
@LappleApple Sorry for being late, I totally missed this one. I added some comments, if we address them we can merge right away.

File: README.md (hunk 3 above)
Comment (szuecs) [inline README.md]: @Raffo I am not sure anymore and this could also change in vault without our notice.

**Comment by Raffo** [comment]
@LappleApple We totally missed this one, didn't we? :-)

File: README.md (hunk 3 above)
Comment (Raffo) [inline README.md]: This is true, but in the end we "document" the vault version we support via Godeps.

**Comment by Raffo** [comment]
LGTM

**Pull request merged**